General/ExportFormats.md

```diff
@@ -0,0 +1,138 @@
+
+It creates one file by class or extented class and works well on windows (compared to file tree).
+
+### Tonel Pros and Cons
+
+Pros
+- Works well on windows 
+- Not much space lost because of small files
+- Export readable files
+- No problem during merge because of metadata
+
+Cons
+- Work out of the box only since Pharo 6.1
+- Since the format is more recent than the others, some edges cases might cause trouble. (It rarely happens)
+
+### Tonel supported Pharo versions
+
+Tonel works out of the box in Pharo 6.1 and higher. But it is possible to make it work in Pharo 4 to 6.0 by updating Metacello and installing Tonel.
+
+```Smalltalk
+Metacello new
+  baseline: 'Metacello';
+  repository: 'github://Metacello/metacello:master/repository';
+  get.
+Metacello new
+  baseline: 'Metacello';
+  repository: 'github://Metacello/metacello:master/repository';
+  onConflict: [:ex | ex allow];
+  load.
+Metacello new 
+	repository: 'github://pharo-vcs/tonel';
+	baseline: 'Tonel';
+	load.
+```
+
+## FileTree metadata less 
+
+FileTree ([https://github.com/dalehenrich/filetree](https://github.com/dalehenrich/filetree)) is the first export format planned to use Pharo with Git that was integrated. It had a first version with a lot of metadata (see section Filetree metadata full) then a new version with a metadata less format. 
+
+This format exports one file per methods. This can cause trouble with Windows because Windows has a maximum path and file name length of 255 characters. This limit can easily be reached with FileTree. 
+
+Another problem of the FileTree format is that most Pharo's method are short and creating one file by method waste a lot of space because the physical representation of files on a drive have a minimal size (4Ko on NTFS file systems) that is reached by most of Pharo's method. 
+
+Its advantage compared to FileTree metadata full is that there is no problem during merge because of metadata.
+
+## FileTree metadata less Pros and Cons 
+
+Pros
+- Easy to browse the history of a method
+- Present since Pharo 4 out of the box
+- No problem during merge because of metadata
+
+Cons
+- Cause trouble on Windows
+- Waste space on the file system
+
+### FileTree metadata less supported Pharo versions
+
+FileTree metadata less is supported out of the box since Pharo 4.0. 
+
+It might be possible to use it in older Pharo version by updating Metacello.
+
+## FileTree metadata full
+
+FileTree metadata full ([https://github.com/dalehenrich/filetree](https://github.com/dalehenrich/filetree))is the ancestor of FileTree metadata less and works in the same way but also export a lot of metadata relatif to the commits such has a commit message (different from the one of the VCS), timestamp of the method compilation, etc.
+
+This simplify the tooling because the tools can get some informations from the metadata instead of the VCS but it also creates a lot of trouble while merging two branches using this format.
```

Comment:
```suggestion
This simplifies the tooling because the tools can get some information from the metadata instead of the VCS but it also creates a lot of trouble while merging two branches using this format.
```